stdlib/grpc/src/main/java/org/ballerinalang/net/grpc/ServicesBuilderUtils.java

```diff
@@ -256,7 +257,7 @@ static BType getBallerinaValueType(BPackage bPackage, String protoType) {
         } else if (protoType.equalsIgnoreCase(WRAPPER_BYTES_MESSAGE)) {
             return new BArrayType(BTypes.typeByte);
         } else {
-            return new BRecordType(protoType, bPackage, 0, true);
+            return new BRecordType(protoType, bPackage, 0, true, TypeFlags.asMask(TypeFlags.ANYDATA));
```

Comment:
Shouldn't we add TypeFlags.PURETYPE too? Since anything that is anydata is pure?

Reply:
Applies in other places too - AbstractSQLStatement.java